src/PowerShellEditorServices/Services/Symbols/IDocumentSymbolProvider.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.PowerShell.EditorServices.Services.Symbols
     /// <summary>
     /// Specifies the contract for a document symbols provider.
     /// </summary>
-    public interface IDocumentSymbolProvider
+    internal interface IDocumentSymbolProvider
```

Comment:
Same as `ICodeLensProvider`, is there another API that allows custom symbols to be registered?

Reply:
Same deal with the lack of registration here